.gitlab-ci.yml

```diff
@@ -1,9 +1,42 @@
 stages:
+  - test
+  - publish
   - veracode scan
   - deps scan
   - generate pages
 
 
+test:
+  stage: test
+  tags:
+    - docker-sock
+  image: advancedtelematic/gitlab-jobs:0.2.0
+  before_script:
+    - docker ps
+    - ./deploy/cleanup.sh
+    - ./deploy/ci_setup.sh
+  script:
+    - docker ps
+    - ./sbt -sbt-dir ./.sbt -ivy ./.ivy2 clean test
+  cache:
+    paths:
+      - .ivy2/
+      - .sbt
+
+publish:
+  stage: publish
+  tags:
+    - docker-sock
+  image: advancedtelematic/gitlab-jobs:0.2.0
+#  only:
+#    - master
+  script:
+    - ./sbt -x -sbt-dir ./.sbt -ivy ./.ivy2 +publish
```

Comment:
same

Reply:
to publish see ota-tuf which already publishes some jars to nexus

Reply:
done